tests/cli.test.js

```diff
@@ -20,6 +21,15 @@ beforeEach(() => {
 	consoleErrSpy = jest.spyOn(console, 'error').mockImplementation();
 });
 
+it('Should be executable via shell', done => {
+	exec("node ./bin/cli.js --version", (error, stdout, stderr) => {
```

Comment:
As mentioned, the problem is somewhere in the release process, because the file in the repo is correct.

Reply:
Then it must be git which is converting. There's nothing else involved.